content/docs/reference/atsign.md

```diff
@@ -13,4 +13,19 @@ toc: false # Change this to "true" to show the table of contents
 weight: 200 # For single pages, lower is first.
 ---
 
-TODO @sign definition goes here
+## What is an @sign
+An @sign is a handle (i.e. @alice) that functions as your digital identity. It uses end-to-end encryption to ensure that your data is 100% owned and controlled by you.
```

Comment:
`i.e.` > `e.g.`

Reply:
I'll fix it! I took that definition from the atsign FAQ's and it shows "i.e." in the Assign FAQ's so it needs to be fixed there as well